packages/utils/src/Logger.ts

```diff
@@ -72,6 +72,8 @@ function wrappedMethodCall(
     }
 }
 
+type Scope = string | { id: string }
```

Comment:
Maybe we could keep using the term _module_? 
```suggestion
type LoggerModule = string | { id: string }
```